README.md

```diff
@@ -89,9 +89,9 @@ In order to provide upgrade path we intend to keep one or more versions of PHP a
 The tag naming strategy consists of (Read as a regex):
 
 - PHP: `(phpMajor).(phpMinor)-(cli|fpm)-(alpine|future supported OSes)(alpineMajor).(alpineMinor)(-dev)?`
-  - Example: `7.2-fpm-alpine3.8`, `7.2-fpm-alpine3.8-dev`
+  - Example: `7.3-fpm-alpine3.10`, `7.3-fpm-alpine3.10-dev`
   - Note: The minor version might come followed by special versioning constraints in case of betas, etc. For instance
-   `7.3-rc-fpm-alpine3.8-dev`
+   `7.4-rc-fpm-alpine3.10-dev`
```

Comment:
We're not building the 7.4-rc, do you want to add it to the matrix?
Let's hope the tests will pass!

Reply:
sure.